src/shared/utilities/childProcess.ts

```diff
@@ -8,6 +8,16 @@
 import * as child_process from 'child_process'
 import * as events from 'events'
 
+export function sanitizeCommand(command: string): string {
```

Comment:
Consider using a module like [`any-shell-escape`](https://www.npmjs.com/package/any-shell-escape) instead of trying to implement our own sanitation logic. It will handle cases like escape characters within `command` that this will miss.